### PR TITLE
Fix dots and slide bug for UICarouselView in web

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: packages/flask/src/UICarouselView/CarouselView/Pagination.tsx [hunk 1]
```diff
@@ -82,5 +82,6 @@ const styles = StyleSheet.create({
         justifyContent: 'space-between',
         alignSelf: 'center',
         alignItems: 'center',
+        height: UIConstant.carousel.circleSize * 2,
```
Comment (savelichalex) [inline packages/flask/src/UICarouselView/CarouselView/Pagination.tsx]: So you're using `height` like some sort of a padding, right?

Comment (ghost) [reply packages/flask/src/UICarouselView/CarouselView/Pagination.tsx]: @savelichalex I just have to pass in the height so the points fit even when scaled. Now the browser calculates the height even from animated points.

**Review by sertony — APPROVED** [review]

**Comment by sertony** [comment]
Can we also change the base into `surf@6.5.0`? I'd like to see this fix in current `release` version of Surf :)

File: packages/flask/src/UICarouselView/CarouselView/Pagination.tsx (hunk 1 above)
Comment (savelichalex) [reply packages/flask/src/UICarouselView/CarouselView/Pagination.tsx]: Sorry, I didn't get it. Could you explain it another way?

Comment (ghost) [reply packages/flask/src/UICarouselView/CarouselView/Pagination.tsx]: @savelichalex Without the specified height, the height is calculated by dots without scaling, therefore the scaled dots do not fit into the view and are cut off. I indicated the height now everything fits.

Comment (savelichalex) [reply packages/flask/src/UICarouselView/CarouselView/Pagination.tsx]: What about increasing it with the same coefficient as you [used for scale](https://github.com/tonlabs/UIKit/blob/a7c1e767451f7252eb599a4875c17580e65fe150/packages/flask/src/UICarouselView/CarouselView/animations.tsx#L82)? Also please leave a comment why it's important to specify the height

Comment (ghost) [reply packages/flask/src/UICarouselView/CarouselView/Pagination.tsx]: @savelichalex done :)

File: packages/flask/src/UICarouselView/CarouselView/Pagination.tsx [hunk 2]
```diff
@@ -82,5 +82,11 @@ const styles = StyleSheet.create({
         justifyContent: 'space-between',
         alignSelf: 'center',
         alignItems: 'center',
+        /**
+         * It is necessary to pass the height of the maximum possible point size.
+         * Otherwise the height will be calculated for the minimum dot size and cut off the scaled dot.
+         * So we have to multiply circleSize by max circle scaling
```
Comment (savelichalex) [inline packages/flask/src/UICarouselView/CarouselView/Pagination.tsx:88]: Great comment 👍

**Review by savelichalex — APPROVED** [review]

File: packages/flask/src/constants/index.ts [hunk 3]
```diff
@@ -52,6 +52,10 @@ export const UIConstant = {
             right: 8,
             bottom: 8,
         },
+        circleScale: {
+            notActive: 1,
+            Active: 1.5,
```
Comment (sertony) [inline packages/flask/src/constants/index.ts]: Can we use a lowercase here to make the "camelCase" be consistent?
```suggestion
            active: 1.5,
```

File: packages/flask/src/UICarouselView/CarouselView/animations.tsx [hunk 4]
```diff
@@ -79,7 +80,10 @@ export const usePaginationStyle = (active: boolean) => {
                     scale: interpolate(
                         animatedValue.value,
                         [PaginationState.NotActive, PaginationState.Active],
-                        [1, 1.5],
+                        [
+                            UIConstant.carousel.circleScale.notActive,
+                            UIConstant.carousel.circleScale.Active,
```
Comment (sertony) [inline packages/flask/src/UICarouselView/CarouselView/animations.tsx]:
```suggestion
                            UIConstant.carousel.circleScale.active,
```

File: packages/flask/src/UICarouselView/CarouselView/Pagination.tsx [hunk 5]
```diff
@@ -82,5 +82,11 @@ const styles = StyleSheet.create({
...
+         */
+        height: UIConstant.carousel.circleSize * UIConstant.carousel.circleScale.Active,
```
Comment (sertony) [inline packages/flask/src/UICarouselView/CarouselView/Pagination.tsx]:
```suggestion
        height: UIConstant.carousel.circleSize * UIConstant.carousel.circleScale.active,
```

**Pull request merged**